configbp/yaml.go

```diff
@@ -0,0 +1,33 @@
+package configbp
+
+import (
+	"fmt"
+	"strconv"
+
+	"gopkg.in/yaml.v2"
+)
+
+// Int64 is an int64 type that can be yaml deserialized from strings.
+//
+// It's useful when the yaml config goes through helm,
+// which could cause precision loss on large int64 numbers:
+// https://github.com/helm/helm/issues/11045
+type Int64 int64
```

Comment:
HelmInt64 or Int64String?

Reply:
Renamed to `YAMLInt64`.

while the bug is on helm, the main difference between this type and vanilla `int64` is on how it handles yaml deserialization, and also it's quite possible that tools other than helm having a similar bug.

Reply:
This is not the "right" way to store int64s in YAML, so I don't like the name YAMLInt64.  `HelmInt64` tells when to use this, and `Int64String` explains that there are strings involved.

Saying "it's also quite possible that other tools that's not helm could have this bug" is a bit specious.  Sure, _all_ code could have a bug, but unless we have evidence that other tools that we use share this specific bug, it seems premature to assume they would.

Reply:
fair, `Int64String` it is.

our log ingestor has this exact same bug, because json is from js, and js only defined a single number type that's float64. and yaml, in some sense, is just an extended json type.